src/helpers/helpers.rtl.js

```diff
@@ -60,16 +60,16 @@ var overrideTextDirection = function(ctx, direction) {
 	}
 };
 
-var restoreTextDirection = function(ctx) {
+const restoreTextDirection = function(ctx) {
 	var original = ctx.prevTextDirection;
 	if (original !== undefined) {
 		delete ctx.prevTextDirection;
 		ctx.canvas.style.setProperty('direction', original[0], original[1]);
 	}
 };
 
-module.exports = {
-	getRtlAdapter: getAdapter,
-	overrideTextDirection: overrideTextDirection,
-	restoreTextDirection: restoreTextDirection,
+export {
```

Comment:
Is there a reason not to `export function getRtlAdapter(..`?

Reply:
Was mostly to keep this minimal, but also because there is a `getRtlAdapter()` already that isn't the one exported